client/src/components/Footer.js

```diff
@@ -1,13 +1,19 @@
 import React from "react";
 import "../pages/Home.css";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
 
 const Footer = () => {
 	return (
 		<footer className="container-fluid">
-			<span>
-				&copy; Code Your Future 2021 | All rights reserved | Registered charity
-				1174929
-			</span>
+			<Container maxWidth="lg">
+				<Typography className="myFooter">
+					<span>
+						&copy; Code Your Future 2021 | All rights reserved | Registered
+						charity 1174929
+					</span>
+				</Typography>
+			</Container>
 		</footer>
```

Comment:
Recreating footer with material-ui  design is a very good idea. Well done Laura!